src/steps/update_horizon_scores.py

```
@@ -1134,6 +1140,30 @@ def extract_task_info(task, headers=None, session=None, project_cache=None):
+
+    Alignment and actionability come from the same LLM call on the same
+    title, so their errors correlate — multiplying them manufactures false
+    precision. Gate on `class` first (what kind of item this is), then rank
+    within the gate using `align` plus small leverage/decay boosts computed
+    in Python, never by the LLM.
+    """
+    align = max(0, min(100, int(align)))
+
+    if cls == "action":
+        return min(100, align + (10 if unblocks else 0) + (10 if expiring else 0))
+    elif cls == "reading":
```

Ruff (0.16.1)

[warning] Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)